### PR TITLE
fix: campaigns,uploads,creatives docs

### PR DESCRIPTION
**Pull request by jho44**
Fixes [ticket](https://lobsters.atlassian.net/browse/DXP-1169)

## Checklist

- [x] Up to date with `main`
- [x] All the tests are passing
  - [x] Delete all resources created in tests
- [x] Prettier
- [x] Spectral Lint
- [x] `npm run bundle` outputs nothing suspect
- [x] `npm run postman` outputs nothing suspect

## Changes
With the help of Andrew Reagan, Jon Harrell, and Julie Lin, the fixes to the Campaigns, Uploads, and Creatives resources are now done. As attributes from `letters` and `postcards` were reused for the Creatives resource, I put their definitions into the corresponding mail piece type's `attributes` folder. 

### Viewing Changes
For those who'd like a convenient way to view the updated docs, just download the [docs file](https://raw.githubusercontent.com/lob/lob-openapi/5f5be63af4ca6192916e03a23b1f1a16ab9642ce/docs/index.html) and point your browser to that path. The styles will be jank but as you scroll down to the actual contents, everything should become legible.

## Areas of Concern (optional)
Neither Julie nor I know:
- what the 422 Validation Error in `Create Uploads` is supposed to be
- why `originalFileName` is a field that can be updated in `Update Uploads`
but these are non-blocking changes.

### CODE REVIEW

**Review by sudoku-lord — APPROVED** [review]

File: resources/creatives/models/creative.yml [hunk 1]
```diff
@@ -30,35 +30,85 @@ allOf:
       template_preview_urls:
         title: Template Preview URLs
         type: object
-        description: URL to view preview of a creative ID. Currently unpopulated, but will be in the future.
+        description: URL to view preview of a creative ID. Currently unpopulated, but will be filled in the future.
```
Comment (julielinter) [inline resources/creatives/models/creative.yml:33]:
```suggestion
        description: preview URLs associated with a creative's artwork asset(s) if the creative uses HTML templates as assets. An empty object will be returned if no `template_preview`s have been generated.
```

File: resources/creatives/models/creative.yml [hunk 2]
```diff
@@ -30,35 +30,85 @@ allOf:
...
       template_previews:
         title: Template Previews
         type: array
         items:
           type: object
-        description: IDs for each creative preview. Currently unpopulated, but will be in the future.
+        description: IDs for each creative preview. Currently unpopulated, but will be filled in the future.
```
Comment (julielinter) [inline resources/creatives/models/creative.yml:39]: Another suggested desc:
```suggestion
A list of template preview objects if the creative uses HTML template(s) as artwork asset(s). An empty array will be returned if no `template_preview`s have been generated for the creative.
```

File: resources/uploads/models/upload_updatable.yml [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
     title: Original Filename
     type: string
-    description: Original filename provided when the upload is created
+    description: Original filename provided when the upload is created.
+  overwriteColumnMapping:
+    title: Overwrite Column Mapping
+    type: boolean
+    description: >
+      Properties in `columnMapping` will be appended to the existing
+      `columnMapping` object.
```
Comment (julielinter) [inline resources/uploads/models/upload_updatable.yml:16]:
```suggestion
```

**Review by julielinter — APPROVED** [review]
I made a few **non-blocking** suggestions, but otherwise, LGTM! Great work @jho44 👍🏻

File: resources/letters/attributes/extra_service.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+type: string
+enum:
+  - certified
+  - certified_return_receipt
+  - registered
+  - null
+description: |
+  Add an extra service to your letter. Can only be non-`null`` if `mail_type` isn't `usps_standard`. See <a href="https://www.lob.com/pricing/print-mail#compare" target="_blank">pricing</a> for extra costs incurred.
```
[the remark this answers is not in the document]
Comment (jho44) [reply resources/letters/attributes/extra_service.yml:8]: No VSCode's autocomplete is trying to end me. Thanks for the catch!

**Pull request merged**